infra/bootstrap/tenant/config/aad-groups/platformAADGroups.defra.json

```diff
@@ -44,6 +44,32 @@
                 ]
             }
         },
+        {
+            "displayName": "AAG-Azure-ADP-Resources-DataReadWrite",
```

Comment:
@rajeshkaremane : As discussed , move these groups to TeamAccessgroups file

Reply:
Updated based on the discussion